mpsl/doc/external_mpsl.rst

```diff
@@ -1,61 +1,65 @@
 .. _external_mpsl:
 
-MPSL
-####
+Library internals
+#################
 
-
-.. toctree::
```

Comment:
Why did you remove the toc?

Reply:
It didn't have any effect when put here. The toctree element is in the parent document (in README.rst) and it pulls this document as a child document under it.

I'm assuming you wanted to use this toctree to show the outline of the sections in this document. I'm not sure if this can be done in this way (possibly yes), but we don't do this in NCS docset, even in much longer documents.

Reply:
Ok, now I understand. No, it's not necessary.